src/Bridge/Symfony/Routing/IriConverter.php

```diff
@@ -94,6 +94,14 @@ public function getItemFromIri(string $iri, array $context = [])
         throw new ItemNotFoundException(sprintf('Item not found for "%s".', $iri));
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getIriFromPlainIdentifier($id, string $resourceClass, array $context = []): string
+    {
+        return $this->getIriFromItem($this->itemDataProvider->getItem($resourceClass, $id, null, $context + ['fetch_data' => true]));
```

Comment:
Like I said on slack, we should use the router to rebuild the route not actually do a request. Code is almost the same as `getIriFromItem`